.dockerignore

```diff
@@ -0,0 +1,8 @@
+# Dockerignore reducing time cost of context upload during docker build for docker/public/Dockerfile-h2o-release.
+# This Dockerfile requires whole H2O folder as context, as it uses multiple files across the project.
+
+# Ignore everything by default 
+*
```

Comment:
Personaly I am against this approach, could we just copy the LICENSE file to build dir via Makefile and and use the build dir still as the base for building the image?

Reply:
Sure, we can. I don't really care about this part. Honoring the current way of doing things is also fine to me.

Done in. https://github.com/h2oai/h2o-3/pull/5197/commits/b7557168ed509d78e84d6492d2cfc3752d8cb2ce